src/blocks/venue/edit.js

```diff
@@ -23,7 +25,7 @@ import { isVenuePostType } from '../../helpers/venue';
 import VenueSelector from '../../components/VenueSelector';
 import VenueInformation from '../../panels/venue-settings/venue-information';
 import OnlineEventLink from '../../components/OnlineEventLink';
-import { Listener } from '../../helpers/broadcasting';
+import { Broadcaster, Listener } from '../../helpers/broadcasting';
```

Comment:
If you'd like to look at the `store` I created for updates to the `Event Date` block, you can start using that instead of the `Listener / Broadcaster`. If not, we can update to that later.

Reply:
> If you'd like to look at the `store` I created for updates to the `Event Date` block, you can start using that instead of the `Listener / Broadcaster`. If not, we can update to that later.

@mauteri Done.